src/prometheus_alert_rules/host_health.rules

```diff
@@ -0,0 +1,25 @@
+groups:
+- name: HostHealth
+  rules:
+  - alert: HostUp
+    expr: up < 1
+    for: 5m
+    labels:
+      severity: warning
+    annotations:
+      summary: Host '{{labels.instance}}' is down.
+      description: >-
+        Host '{{ $labels.instance }}' is down.
+          VALUE = {{ $value }}
+          LABELS = {{ $labels }}
+  - alert: HostAvailable
+    expr: absent(up)
+    for: 5m
+    labels:
+      severity: warning
+    annotations:
+      summary: Metrics endpoint for host '{{labels.instance}}' is unreachable.
```

Comment:
Iiuc, we can be certain that the metrics endpoint for host in unreachable when `up < 1` (the first alert).
For `absent`, how about:
- No recent data from host
- Host data not received
- Data from host overdue